<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profil</class>
 <widget class="QDialog" name="profil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(224, 224, 224);</string>
  </property>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Cherecher MÃ©dicament</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouter_Vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Ajouter Vente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:20px;
font: 20pt &quot;Segoe Script&quot;;color:rgb(170, 85, 255)</string>
   </property>
   <property name="text">
    <string>PHARMACIE N.M.B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/OIP (3).jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;font-size:15px;border:2px solid black;color:white;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_afficher_vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Consulter les ventes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>1011</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:15px;
font: 600 16pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Dr.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonProfil">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nome_user">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
font: 14pt &quot;Segoe Fluent Icons&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Consulter Stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Rockwell&quot;;
background-color: rgb(255, 255, 255);
border:2px solid rgb(235, 15, 255);
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>     Votre Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>721</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_nom">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_prenom">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numero">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_etat">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_identifiant">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>  Numero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>    Etat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>  Identifiant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>    Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 12pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>    Email</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_GotoModifier">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>560</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 20pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(96, 156, 18);;color:white;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Annulerconsulter">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 125, 218);color:white;border-radius:10px;
font: 900 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/WhatsApp Image 2024-04-30 Ã  20.55.44_6b270705.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue;border-radius:4px;
font: 14pt &quot;Rockwell&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/devenir-pharmacien-921x1024.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>line_4</zorder>
  <zorder>line</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_ajouter_Vente</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_afficher_vente</zorder>
  <zorder>label_6</zorder>
  <zorder>ButtonProfil</zorder>
  <zorder>line_2</zorder>
  <zorder>nome_user</zorder>
  <zorder>pushButton_stock</zorder>
  <zorder>label_8</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_GotoModifier</zorder>
  <zorder>pushButton_Annulerconsulter</zorder>
  <zorder>label_17</zorder>
  <zorder>label_date</zorder>
  <zorder>label_15</zorder>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
